scanner.mll

```diff
@@ -37,10 +37,22 @@ rule tokenize = parse
 | "false" { BLIT(false) }
 (* 2.6 operators *)
 | '='        { ASSIGN }
+| "+="       { PLUSASSIGN }
+| "-="       { MINUSASSIGN }
```

Comment:
Are we planning on adding more, such as `*=` ?

Reply:
We don't have such a plan so far.